src/test/kotlin/com/vanniktech/rxriddles/solutions/Riddle37Solution.kt

```diff
@@ -0,0 +1,12 @@
+package com.vanniktech.rxriddles.solutions
+
+import io.reactivex.Observable
+import java.io.IOException
+
+object Riddle37Solution {
+  fun solve(source: Observable<Boolean>): Observable<Boolean> {
```

Comment:
can you remove the return type and have the same styling as the other solutions?